app/src/main/java/org/p2p/wallet/user/interactor/UserTokensInteractor.kt

```diff
@@ -68,4 +73,11 @@ class UserTokensInteractor(
     suspend fun getUserBalance(): BigDecimal {
         return userTokensLocalRepository.getUserBalance()
     }
+
+    suspend fun setTokenHidden(mintAddress: String, visibility: String) =
+        userTokensLocalRepository.setTokenHidden(mintAddress, visibility)
+
+    suspend fun clear() {
+        userTokensLocalRepository.clear()
```

Comment:
we clear repositories, but not interactors, it's better to inject userTokensLocalRepository and clear it